src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -1,22 +1,24 @@
 package frc.robot.subsystems;
 
 import frc.robot.Constants;
-
+import frc.robot.RobotContainer;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import edu.wpi.first.wpilibj.DigitalInput;
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import frc.robot.commands.*;
 
 //CANSpark imports
 import com.revrobotics.CANSparkMax;
 import com.revrobotics.CANSparkMaxLowLevel.MotorType;
 import com.revrobotics.RelativeEncoder;
+import com.revrobotics.CANSparkMax.SoftLimitDirection;
 
 public class IntakeSubsystem extends SubsystemBase {
 
   // Defines Intake Motor Variables
   // Uncomment intake Acuators when we write that code
-  private CANSparkMax intakeLeftActuator;
-  private CANSparkMax intakeRightActuator;
+  public CANSparkMax intakeLeftActuator;
+  public CANSparkMax intakeRightActuator;
```

Comment:
These should be private, not public.  If any code outside of this subsystem need to read or write the actuators then create getter and setter methods.